libs/utils/utils.js

```diff
@@ -86,6 +87,7 @@ const AUTO_BLOCKS = [
 const ENVS = {
   local: {
     name: 'local',
+    adobeIO: 'cc-collab.adobe.io',
```

Comment:
We use the stage edge config & pdfViewer, should we just copy all the stage configs to local?
It would also make sense for ims etc. to properly test locally using stage data

Reply:
This was included by mistake and I will delete.